app/controllers/conversations_controller.rb

```diff
@@ -118,4 +121,11 @@ def contacts_data
         {value: contact_id, name: ERB::Util.h(Person.name_from_attrs(*name_attrs)) }
       }
   end
+
+  def no_contacts?
```

Comment:
Can you make that a bit more explicit? `not_admin_and_no_contacts?` for example?

Reply:
"no_contacts_for_user" / "no_contacts_for_users_role"? .. For admin seems to specialized fo me for a methods' name?